MatrixKit/Models/Room/MXKRoomBubbleCellDataStoring.h

```diff
@@ -121,20 +121,25 @@
 @property (nonatomic) BOOL isTyping;
 
 /**
- Show the date time label in rendered bubble cell (NO by default)
+ Show the date time label in rendered bubble cell. NO by default.
  */
 @property (nonatomic) BOOL showBubbleDateTime;
 
 /**
- The date time label is not managed by MatrixKit. (NO by default).
+ Disable the default display of date time labels by MatrixKit. NO by default.
  */
 @property (nonatomic) BOOL useCustomDateTimeLabel;
```

Comment:
Weird
-> Disable the default ..
The member is named useCustomDateTimeLabel

Show the dis... seems better